index.html

```diff
@@ -2133,12 +2142,17 @@ <h3>Algorithm</h3>
                   <var>value</var> is not a <a>string</a>, an
                   <a data-link-for="JsonLdErrorCode">invalid language-tagged string</a>
                   error has been detected and processing is aborted.
+                  <span class="changed">If <var>value</var> is not <a>well-formed</a> according to
```

Comment:
```suggestion
                  <span class="changed">Otherwise if <var>value</var> is not <a>well-formed</a> according to
```
This is require so that the "Otherwise" below (line 2149) still makes sense.

Reply:
Actually, I think the clauses should be inverted, so the Otherwise has to do with setting expanded value to value, and do the warning later.

> If expanded property is `@language` and value is not a string, an **invalid language-tagged string** error has been detected and processing is aborted, _otherwise, set `expanded value` to `value`_. If `value` is not well-formed according to section 2.2.9 of [BCP47], processors SHOULD issue a warning.  ...

Reply:
Fixed in 1c62284b358db33cab889fbff62aee2558dce78c.